assets/section-main-blog.css

```diff
@@ -59,3 +59,35 @@
     padding-bottom: 82.5rem;
   }
 }
+
+.blog--collapsed-y:not(.contains-card--standard) .card-wrapper {
```

Comment:
For now and from what I'm seeing `.contains-card-standard` isn't ever applied on that element but we might need to add it.